code/game/gamemodes/objective.dm

```diff
@@ -126,6 +126,7 @@ GLOBAL_LIST_EMPTY(objectives) //SKYRAT EDIT ADDITION
 	return target
 
 //dupe_search_range is a list of antag datums / minds / teams
+//optinlevel is our list of people who have opted in, if you're not on that you're not a valid target
```

Comment:
this comment is irrelevant now btw

Reply:
If it's irrelevant, then let's remove it, yeah?
```suggestion
```